poetry/console/commands/export.py

```diff
@@ -29,7 +29,11 @@ def handle(self):
         fmt = self.option("format")
 
         if fmt not in Exporter.ACCEPTED_FORMATS:
-            raise ValueError("Invalid export format: {}".format(fmt))
+            raise ValueError(
+                "Invalid export format: {} valid formats: {}".format(
```

Comment:
Thanks for your PR! 🙌

Adding a newline in between would be better for readability.